cegs_portal/search/templates/search/v1/search_results.html

```diff
@@ -294,11 +440,32 @@
 
         updateFacetState();
 
+        let queryParam = new URLSearchParams(window.location.search);
+        let searchFeatures = queryParam.getAll("feature_type");
+        featureCheckboxes.forEach(checkbox => {
+            checkbox.checked = searchFeatures.includes(checkbox.id);
+            checkbox.addEventListener("change", _ => {
+                updateFeatureState();
+                let region = state.g(STATE_REGION);
+                let featureTypes = state.g(STATE_FEATURE_TYPES);
+                let url = `/search/v1/featureloc/chr${region.chr}/${region.start}/${region.end}`;
+
+                if (featureTypes.length > 0) {
+                    url += `?feature_type=${featureTypes.join('&feature_type=')}`;
+                }
+
+                htmx.ajax('GET', url, {target: '#features-table'});
+                        });
```

Comment:
This should all be part of a `state.addCallback(STATE_FEATURE_TYPES, (s, key) => { });` block. The flow should be like this: user clicks on a feature -> updateFeatureState() -> all `state.addCallback(STATE_FEATURE_TYPES...` callbacks get called. Look at the code involved in the `STATE_FACETS` state to get an idea of this.